Team8923_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team8923_CENTERSTAGE/BaseOpMode.java

```diff
@@ -46,9 +71,16 @@ public void initHardware() {
 
         motorFL.setDirection(DcMotor.Direction.FORWARD);
         motorFR.setDirection(DcMotor.Direction.REVERSE);
-        motorBL.setDirection(DcMotor.Direction.REVERSE);
+        motorBL.setDirection(DcMotor.Direction.FORWARD);
         motorBR.setDirection(DcMotor.Direction.REVERSE);
 
+        motorIntakeWheels.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        motorIntakeWheels.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        motorIntakeWheels.setDirection(DcMotor.Direction.FORWARD);
+
+        servoFlipGondola.setPosition(0.25);
+
+        servoRotateGondola.setPosition(0.0);
```

Comment:
You could maybe have start position constants.

Reply:
Good point, I have added constants for initialized constants!